tuichain/api/views/loanrequests.py

```diff
@@ -152,7 +152,7 @@ def get_non_validated_loan_requests(request):
     Get all active and non_validated loan requests
     """
 
-    loanrequest_list = LoanRequest.objects.filter(active=True, validated=False)
+    loanrequest_list = LoanRequest.objects.filter(status=1)
```

Comment:
Souldn't the status be 0 here?

Reply:
Yes it should I think because 0 - pending and 1 is funding already.

Reply:
Yes, it should
Anyways, that function will not be included in the next commit